README.md

```diff
@@ -19,9 +19,35 @@ To create a new project with the tool:
 apostrophe create <project name>
 ```
 
-If you run the `create` command with the `--install` flag, the command will also `npm install` the dependencies of the project and `apostrophe:reset` the database, resulting in a fully bootstrapped project.
+This will create a local copy of our standard [Apostrophe Boilerplate](https://github.com/punkave/apostrophe-boilerplate).
 
-#### Run command-line tasks
+If you run the `create` command with the `--install` flag, the command will also `npm install` the dependencies for the project and `apostrophe:reset` the database, resulting in a fully bootstrapped project.
+
+#### Create a widget
+To bootstrap the necessary files and basic configuration for a new Apostrophe widget, run the following command from within your Apostrophe project's root directory:
+```bash
+apostrophe createWidget <widgetName>
```

Comment:
Include an example so people know what you expect them to type here. The widget name vs. the full module name...